types/tailwindcss/resolveConfig.d.ts

```diff
@@ -0,0 +1,5 @@
+import type { TailwindConfig } from './tailwind-config';
+
+declare function resolveConfig(tailwindConfig: TailwindConfig): TailwindConfig;
+
+export default resolveConfig;
```

Comment:
```suggestion
export = resolveConfig;
```

Reply:
It does seem odd that this one is `export default` while the others are all `export =`.

Reply:
@dolanmiu

Reply:
@amcasey done, the change is applied

Reply:
I have reverted this change as mentioned as it broke the tests